07-sampling.Rmd

```diff
@@ -1064,33 +1062,245 @@ Comment on the representativeness of the following *sampling methodologies*:
+It is useful to represent a trial as a random variable that takes the value of 1 if the ball is red or the value of 0 if the ball is not red. 
+We denote the first trial $X_1$, the second trial $X_2$, and so on. 
+The average of these 25 trials, before they are carried out, can also be expressed as a random variable: $$\bar X = \frac{X_1+X_2+\dots+X_{25}}{25}.$$
+Note that $\bar X$ is a random variable before the trials are run, so $\bar X$ represents all the possible sample proportions that we could obtain. After we run the first trial, if the ball observed is red, we assign $X_1 = 1$, and if it is not red $X_1=0$. Once all 25 trials have been performed, we get a list of 25 ones or zeros. As an illustration, one possible outcome could be: $$\{0,0,0,1,0,1,0,1,0,0,1,0,0,1,1,0,0,0,1,1,0,1,0,0,0,1 \}$$  The observed value of $\bar X$ will be 
+$$\bar X = \frac{0+0+0+1+\dots+0+1}{25} = \frac{10}{25}=0.4.$$ 
+which is both the sample mean of those trial runs and the sample proportion of red balls in the observed sample. 
+Sample proportions are sample means!
 
-In other words, as our sample size gets larger (1) the sampling distribution of a point estimate (like a sample proportion) increasingly follows a *normal distribution* and (2) the variation of these sampling distributions gets smaller, as quantified by their standard errors. We discuss the properties of the normal distribution in Appendix \@ref(appendix-normal-curve).
 
-Shuyi Chiou, Casey Dunn, and Pathikrit Bhattacharyya created a 3-minute and 38-second video at <https://youtu.be/jvoxEYmQHNM> explaining this crucial statistical theorem using the average weight of wild bunny rabbits and the average wingspan of dragons as examples. Figure \@ref(fig:CLT-video-preview) shows a preview of this video.
 
-```{r CLT-video-preview, echo=FALSE, fig.cap="Preview of Central Limit Theorem video.", purl=FALSE, out.width = "75%"}
-knitr::include_graphics("images/copyright/CLT_video_preview.png")
+### The distribution of $\bar X$
+
+The sampling distribution of the sample proportion (or sample mean) is now represented by the distribution of $\bar X$. Think of this distribution as how often each of the values that $\bar X$ could take may end up happening. 
+
+A very useful way to learn about this distribution is using simulations. 
+Simulations seldom provide the exact structure of the distribution, because we would need an infinite number of samples, but a large number of replications often produce a really good approximation of the distribution and we can use it to study the distribution's characteristics. We now repeat our experiment 10,000 times. 
```

Comment:
```suggestion
Simulations seldom provide the exact structure of the distribution, because we would need an infinite number of samples to get the exact layout. A large number of replications can often produce a really good approximation of the exact distribution, and we can use it to study the distribution's characteristics. We now repeat our experiment 10,000 times. 
```